src/components/button/styled.tsx

```diff
@@ -150,7 +142,7 @@ export const StyledButton = styled(({ label, icon, ...otherProps }) => (
     &button {
       padding: 0;
     }
-    padding: 8px;
+    padding: ${getSizeBy(1)};
```

Comment:
i don't understand why this is an improvement, is there a possibility that we will be changing all sizes in the application by changing `SIZE_UNIT`? Correct me if i'm wrong, but i think it's very unlikely. If we really need wrapper, perhaps we could at least use the same number, without having to multiply/divide by 8 all the time, when comparing this to designs?

Reply:
could be just `getSizeBy()`

Reply:
It's a consistent practice that allows to stick with defined measurements. Any wrong size in the mock is rounded to agreed measurements, unless it makes no sense (small icons etc). So, even if we won't change the base size unit ever, I believe it's a nice practice to add more consistency to the layouts.

Reply:
thx for explanation, it wasn't clear to me that you agreed to stick with multiplicities of 8 and that is the purpose of those functions. It just looked a little over-engineered (IMHO), a little complication to trivial "px" value, but of course there's no problem to getting used to it :+1:. Oh, and personally i prefer explicit (1) than a default number, just one less thing to worry about